src/main/java/uk/gov/companieshouse/service/DissolutionValidator.java

```diff
@@ -19,12 +20,16 @@ public DissolutionValidator(CompanyProfileService companyProfileService, Company
         this.companyOfficerService = companyOfficerService;
     }
 
-    public Optional<String> checkBusinessRules(CompanyProfileApi companyProfileApi) {
+    public Optional<String> checkBusinessRules(
+            CompanyProfileApi companyProfileApi,
+            List<DirectorRequest> selectedDirectors) {
         if (!companyProfileService.isCompanyClosable(companyProfileApi)) {
             return Optional.of("Company must be of a closable type and have an active status");
         }
 
-        if (!companyOfficerService.hasEnoughOfficersSelected(companyProfileApi.getCompanyNumber())) {
+        if (!companyOfficerService.hasEnoughOfficersSelected(
+                companyProfileApi.getCompanyNumber(),
+                selectedDirectors)) {
```

Comment:
i don't know about you but i find this so hard to read compared to:
```
if (!companyOfficerService.hasEnoughOfficersSelected(companyProfileApi.getCompanyNumber(), selectedDirectors)) {
```

Reply:
that's fair,
I entered the line breaks for line length reasons>

Im not partial to them so ill get rid